quick-github-guide.md

```diff
@@ -0,0 +1,163 @@
+# Quick guide to working with a GitHub repo
```

Comment:
Suggestion: The Google Developer Docs style guide recommends against the repo abbreviation. Please consider replacing repo with repository or defining repo as an abbreviation of repository.

Reply:
Done.